readMe.md

```diff
@@ -1,8 +1,11 @@
 ### Doppelkopf
 
-All core functionality can be found under src/core/...
+All core functionality can be found under src/core/...   
 The main game loop is in src/main
 
+### Test
+Test compilation is off by default, to activate it pass `-DADD_TESTS=ON` to cmake when loading the cmake project(for ex. in the gui)
+

```

Comment:
It should be added here that you need  `git submodule init` and `git submodule update` to install the gtest suite